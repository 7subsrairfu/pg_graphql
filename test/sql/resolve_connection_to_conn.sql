select jsonb_pretty(
    gql.resolve($$
{
  allAccounts {
    edges {
      node {
        id
        email
        blogs {
          totalCount
            edges {
              node {
                name
            }
          }
        }
      }
    }
  }
}
    $$)
);
