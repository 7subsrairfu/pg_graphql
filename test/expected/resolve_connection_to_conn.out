select jsonb_pretty(
    gql.resolve($$
{
  allAccounts {
    edges {
      node {
        id
        email
        blogs {
          totalCount
            edges {
              node {
                name
            }
          }
        }
      }
    }
  }
}
    $$)
);
                        jsonb_pretty                         
-------------------------------------------------------------
 {                                                          +
     "data": {                                              +
         "allAccounts": {                                   +
             "edges": [                                     +
                 {                                          +
                     "node": {                              +
                         "id": 1,                           +
                         "blogs": {                         +
                             "edges": [                     +
                                 {                          +
                                     "node": {              +
                                         "name": "A: Blog 1"+
                                     }                      +
                                 },                         +
                                 {                          +
                                     "node": {              +
                                         "name": "A: Blog 2"+
                                     }                      +
                                 },                         +
                                 {                          +
                                     "node": {              +
                                         "name": "A: Blog 3"+
                                     }                      +
                                 }                          +
                             ],                             +
                             "totalCount": 3                +
                         },                                 +
                         "email": "aardvark@x.com"          +
                     }                                      +
                 },                                         +
                 {                                          +
                     "node": {                              +
                         "id": 2,                           +
                         "blogs": {                         +
                             "edges": [                     +
                                 {                          +
                                     "node": {              +
                                         "name": "B: Blog 3"+
                                     }                      +
                                 }                          +
                             ],                             +
                             "totalCount": 1                +
                         },                                 +
                         "email": "bat@x.com"               +
                     }                                      +
                 },                                         +
                 {                                          +
                     "node": {                              +
                         "id": 3,                           +
                         "blogs": {                         +
                             "edges": null,                 +
                             "totalCount": 0                +
                         },                                 +
                         "email": "cat@x.com"               +
                     }                                      +
                 },                                         +
                 {                                          +
                     "node": {                              +
                         "id": 4,                           +
                         "blogs": {                         +
                             "edges": null,                 +
                             "totalCount": 0                +
                         },                                 +
                         "email": "dog@x.com"               +
                     }                                      +
                 },                                         +
                 {                                          +
                     "node": {                              +
                         "id": 5,                           +
                         "blogs": {                         +
                             "edges": null,                 +
                             "totalCount": 0                +
                         },                                 +
                         "email": "elephant@x.com"          +
                     }                                      +
                 }                                          +
             ]                                              +
         }                                                  +
     },                                                     +
     "errors": [                                            +
     ]                                                      +
 }
(1 row)

